web-api/src/personalization/filter.rs

```diff
@@ -288,6 +308,74 @@ impl Filter {
             Self::Combine(combine) => combine.filters.is_below_depth(max),
         }
     }
+
+    pub(crate) fn insert_published_after(
```

Comment:
we only need to keep this around until the deprecated request option gets removed.